Source/WebCore/html/EmailInputType.cpp

```diff
@@ -88,6 +91,14 @@ bool EmailInputType::supportsSelectionAPI() const
     return false;
 }
 
+void EmailInputType::attributeChanged(const QualifiedName& name)
+{
+    if (name == multipleAttr)
+        element()->setValueFromRenderer(sanitizeValue(element()->value()));
+
+    InputType::attributeChanged(name);
```

Comment:
This should call the base class's implementation, which would be BaseTextInputType, not InputType